Tracks Properties/Add 1db to selected tracks sends via send window (or fader volume if no send window)/Subtract 1db from selected tracks sends via send window (or fader volume if no send window).lua

```diff
@@ -0,0 +1,308 @@
+scriptName = "Subtract 1db from selected tracks sends via send window (or fader volume if no send window)"
```

Comment:
-- @description Adjust selected tracks sends via send window (or fader volume if no send window)
-- @author Triode
-- @version 1
-- @metapackage
-- @provides [main] Add 1db to selected tracks sends via send window (or fader volume if no send window)/Subtract 1db from selected tracks sends via send window (or fader volume if no send window).lua > Add 1db to selected tracks sends via send window (or fader volume if no send window)/Subtract 1db from selected tracks sends via send window (or fader volume if no send window)
-- @donation Donate via PayPal https://www.paypal.me/outoftheboxsounds
-- @about
--   This script will adjust the relative level of all the selected tracks' sends named in an open REAPER send window.
--   If no send window is open, the selected tracks fader volume will be adjusted.
--
--   You can edit the db value that the script uses inside the script.
--
--   It works on regular tracks as well as the master track and can target sends as well as hardware outs and will distinguish between mono and stereo sends.
--
--   It requires  js_ReaScriptAPI extension by juliansader
--
--   Thanks also to Edgemeal for the code at the start of this script and Bfooz for some of the send code.